src/main/java/org/opensearch/ad/transport/AnomalyResultTransportAction.java

```diff
@@ -348,19 +349,17 @@ public void onResponse(CompositeRetriever.Page entityFeatures) {
                                 continue;
                             }
                             String modelNodeId = modelNode.getId();
-                            if (stateManager.isMuted(modelNodeId)) {
-                                LOG.info(String.format(Locale.ROOT, NODE_UNRESPONSIVE_ERR_MSG + " %s", modelNodeId));
+                            if (stateManager.isMuted(modelNodeId, detectorId)) {
+                                LOG.info(String.format(Locale.ROOT, NODE_UNRESPONSIVE_ERR_MSG + " %s for %s", modelNodeId, detectorId));
```

Comment:
minor - maybe add some wording to distinguish the node ID vs. detector ID?

Something like:
```
LOG.info(String.format(Locale.ROOT, NODE_UNRESPONSIVE_ERR_MSG + " %s for detector %s", modelNodeId, detectorId));
```

Reply:
sure, changed